lib/strings.nix

```diff
@@ -674,6 +675,25 @@ rec {
     else
       false;
 
+  /* Parse a string as a float.
+
+     Type: string -> float
+
+     Example:
+       toFloat "1.2"
+       => 1.2
+       toFloat "-4.2"
+       => -4
+       toFloat "1"
+       => error: Could not convert 1 to float.
+  */
+  # Obviously, it is a bit hacky to use fromJSON this way.
+  toFloat = str:
```

Comment:
I'm not sure I'm qualified to approve this, although I guess it seems fine?

Reply:
@Infinisil (or somebody else), could you maybe approve this addition to `lib/strings.nix`